### PR TITLE
taxonomy: Labels

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: Labels

### CODE REVIEW

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -3904,11 +4081,29 @@ de:Enthält Alkohol
...
 pt:Contém álcool
+es:Contiene alcohol
+fr:Contient de l'alcool
+nl:Bevat alcohol
+pl:Zawiera alkohol
+ru:Содержит алкоголь
 opposite:en: en:no-alcohol
 
 en:without allergens
 hr:sans allergènes
```
Comment (benbenben2) [inline taxonomies/labels.txt:4092]:
```suggestion
```

**Review by benbenben2 — APPROVED** [review]
Looks good to me

File: taxonomies/labels.txt [hunk 2]
```diff
@@ -21902,12 +22324,23 @@ fi:Kanat kasvatettu ilman antibiootteja
...
 <en:Chickens raised without antibiotics
 en:Chickens raised without antibiotics since birth
 fr:Poulets élevés sans antibiotique dès la naissance
 pt:Galinha criada sem antibióticos desde o nascimento, galinhas criadas sem antibióticos desde o nascimento
+bg:Пилета отглеждани без антибиотици от раждането
+ca:Gallines criades sense antibiòtics des de la naixença
+de:Hühneraufzucht ohne Antibiotika seit der Geburt
+es:Pollos criados sin antibióticos desde el nacimiento
+fi:Kanat kasvatettu ilman antibiootteja syntymästä asti
+hu:Antibiotikumok nélkül nevelt csirkék
```
Comment (benbenben2) [inline taxonomies/labels.txt:22342]:
```suggestion
```

File: taxonomies/labels.txt [hunk 3]
```diff
@@ -21716,6 +22126,10 @@ origins:en: en:france
 ingredients:en: en:potato
 
 fr:Pommes de terre origine France
+es:Patatas de origen Francia
+de:Kartoffeln aus Frankreich
```
Comment (benbenben2) [inline taxonomies/labels.txt:22130]:
```suggestion
```

File: taxonomies/labels.txt [hunk 4]
```diff
@@ -4204,10 +4434,39 @@ bg:Ниско съдържание на наситени мазнини
...
+th:ปริมาณไขมันอิ่มตัวต่ำ
+tr:düşük doymuş yağ içeriği
+zh:低饱和脂肪含量
 description:en:In EU, a claim that a food is low in saturated fat may only be made if the sum of saturated fatty acids and trans-fatty acids in the product does not exceed 1,5 g per 100 g for solids or 0,75 g/100 ml for liquids and in either case the sum of saturated fatty acids and trans-fatty acids must not provide more than 10 % of energy. (see: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02006R1924-20141213)
 
 en:saturated fat-free
 xx:saturated fat-free
+bg:без наситени мазнини
+ca:sense greixos saturats
+de:fettfrei, ohne gesättigte Fettsäuren
```
Comment (benbenben2) [inline taxonomies/labels.txt:4456]:
```suggestion
de:ohne gesättigte Fettsäuren
```

File: taxonomies/labels.txt [hunk 5]
```diff
@@ -4204,10 +4434,39 @@ bg:Ниско съдържание на наситени мазнини
...
+bg:без наситени мазнини
+ca:sense greixos saturats
+de:ohne gesättigte Fettsäuren
+es:sin grasas saturadas
+fi:täysin kyllästetty rasvaton
+fr:sans matières grasses saturées
+hr:bez zasićenih masti
+hu:telített zsírmentes
+it:senza grassi saturi
+nl:vetvrij, zonder verzadigde vetzuren
```
Comment (benbenben2) [inline taxonomies/labels.txt:4463]:
```suggestion
nl:zonder verzadigde vetzuren
```

**Comment by teolemon** [comment]
Errors in the labels taxonomy definition:
ERROR - nl:rijk-aan-ijzer already is a synonym of nl:bron-van-ijzer for entry en:iron-source (labels) - nl:rijk-aan-ijzer cannot be mapped to entry en:rich-in-iron / nl:rijk-aan-ijzer
ERROR - bg:произведено-в-европейския-съюз already is a synonym of bg:произведено-в-ес for entry en:made-in-the-eu (labels) - bg:произведено-в-европейския-съюз cannot be mapped to entry bg:произведено-в-европейския-съюз / bg:произведено-в-европейския-съюз
ERROR - ca:fet-a-la-unio-europea already is a synonym of ca:fet-a-la-ue for entry en:made-in-the-eu (labels) - ca:fet-a-la-unio-europea cannot be mapped to entry bg:произведено-в-европейския-съюз / ca:fet-a-la-unio-europea
ERROR - cs:vyrobeno-v-eu already is associated to en:made-in-the-eu (labels) - cs:vyrobeno-v-eu cannot be mapped to entry bg:произведено-в-европейския-съюз
ERROR - de:hergestellt-in-der-eu already is associated to en:made-in-the-eu (labels) - de:hergestellt-in-der-eu cannot be mapped to entry bg:произведено-в-европейския-съюз
ERROR - es:hecho-en-la-ue already is associated to en:made-in-the-eu (labels) - es:hecho-en-la-ue cannot be mapped to entry bg:произведено-в-европейския-съюз
ERROR - fi:valmistettu-eu-ssa already is associated to en:made-in-the-eu (labels) - fi:valmistettu-eu-ssa cannot be mapped to entry bg:произведено-в-европейския-съюз
ERROR - fr:fabrique-dans-l-ue already is associated to en:made-in-the-eu (labels) - fr:fabrique-dans-l-ue cannot be mapped to entry bg:произведено-в-европейския-съюз
ERROR - hr:proizvedeno-u-europskoj-uniji already is a synonym of hr:proizvedeno-u-eu for entry en:made-in-the-eu (labels) - hr:proizvedeno-u-europskoj-uniji cannot be mapped to entry bg:произведено-в-европейския-съюз / hr:proizvedeno-u-europskoj-uniji
ERROR - it:prodotto-nell-unione-europea already is a synonym of it:prodotto-nell-ue for entry en:made-in-the-eu (labels) - it:prodotto-nell-unione-europea cannot be mapped to entry bg:произведено-в-европейския-съюз / it:prodotto-nell-eu
ERROR - pl:wyprodukowano-w-ue already is associated to en:made-in-the-eu (labels) - pl:wyprodukowano-w-ue cannot be mapped to entry bg:произведено-в-европейския-съюз
ERROR - pt:feito-na-ue already is associated to en:made-in-the-eu (labels) - pt:feito-na-ue cannot be mapped to entry bg:произведено-в-европейския-съюз
ERROR - bg:без-изкуствени-добавки already is associated to en:no-artificial-additives (labels) - bg:без-изкуствени-добавки cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - ca:sense-additius-artificials already is associated to en:no-artificial-additives (labels) - ca:sense-additius-artificials cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - de:ohne-künstliche-zusätze already is associated to en:no-artificial-additives (labels) - de:ohne-künstliche-zusätze cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - es:sin-aditivos-artificiales already is associated to en:no-artificial-additives (labels) - es:sin-aditivos-artificiales cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - fi:ei-keinotekoisia-lisäaineita already is associated to en:no-artificial-additives (labels) - fi:ei-keinotekoisia-lisäaineita cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - fr:sans-additifs-artificiels already is associated to en:no-artificial-additives (labels) - fr:sans-additifs-artificiels cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - he:ללא-תוספים-מלאכותיים already is associated to en:no-artificial-additives (labels) - he:ללא-תוספים-מלאכותיים cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - it:senza-additivi-artificiali already is associated to en:no-artificial-additives (labels) - it:senza-additivi-artificiali cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - nl:zonder-kunstmatige-toevoegingen already is associated to en:no-artificial-additives (labels) - nl:zonder-kunstmatige-toevoegingen cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - pt:sem-aditivos-artificiais already is associated to en:no-artificial-additives (labels) - pt:sem-aditivos-artificiais cannot be mapped to entry bg:без-изкуствени-добавки
ERROR - pl:bez-sztucznych-barwników-i-konserwantów already is associated to en:no-artificial-colours-and-preservatives (labels) - pl:bez-sztucznych-barwników-i-konserwantów cannot be mapped to entry en:no-artificial-colours-or-flavours

**Review by benbenben2 — APPROVED** [review]
LGTM

Great work @teolemon!

**Pull request merged**